druid_setup/environment

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
Druid recommends splitting out the environment files, do we want to do that? https://druid.apache.org/docs/latest/tutorials/docker.html#environment-file